phoronix/run_phoronix.sh

```diff
@@ -118,35 +124,6 @@ if [ $show_usage -eq 1 ]; then
 	usage $0
 fi
 
-test_tools/package_tool --update
-test_tools/package_tool --packages php73-cli.x86_64,php73-common.x86_64,php73-xml.x86_64
```

Comment:
If we are removing this, we need to update the phoronix test config file to get them installed on test installation.